test/e2e/data/infrastructure-azure/v1.11.7/cluster-template-prow.yaml

```diff
@@ -57,27 +57,15 @@ spec:
     clusterConfiguration:
       apiServer:
         extraArgs:
-          cloud-config: /etc/kubernetes/azure.json
-          cloud-provider: azure
+          cloud-provider: external
```

Comment:
What accounts for the changes to this template from in-tree to external cloud-provider?

Reply:
We don't want to be creating any in-tree cloud-provider clusters anymore since it soon won't be supported at all. This was one of the leftover templates that still used in-tree since it created clusters with an older version of k8s for API version upgrade tests but since #4433 merged we can use the same template as other v1beta1 cluster tests (I deleted the template altogether in my latest commit).